sycl/test/basic_tests/set_arg_error.cpp

```diff
@@ -33,15 +33,23 @@ int main() {
     cl::sycl::accessor<int, 1, cl::sycl::access::mode::read_write,
                        cl::sycl::access::target::local>
         local_acc({size}, h);
+    TriviallyCopyable tc{1, 2};
+    NonTriviallyCopyable ntc;
     h.set_arg(0, local_acc);
     h.set_arg(1, global_acc);
     h.set_arg(2, samp);
-    h.set_arg(3, TriviallyCopyable{});
+    h.set_arg(3, tc);
+    h.set_arg(4, TriviallyCopyable{});
+    h.set_arg( // expected-error {{no matching member function for call to 'set_arg'}}
+        5, ntc);
     h.set_arg( // expected-error {{no matching member function for call to 'set_arg'}}
         4, NonTriviallyCopyable{});
```

Comment:
```suggestion
        6, NonTriviallyCopyable{});
```

Reply:
I don't think that this change is necessary because this test just checks for compile-time errors. So these numbers can be anything.

Reply:
I'm agree with you. It is just an optional suggestion.

Reply:
@KarachunIvan, feel free to apply this "unnecessary" change in a separate PR. I'm merging this one.